src/LayerEditor/ui/diagram_editor/graphics_items/gs_point.py

```diff
@@ -46,9 +51,22 @@ def __init__(self, pt):
         self.setAcceptedMouseButtons(QtCore.Qt.LeftButton | QtCore.Qt.RightButton)
         self.update()
 
+    def init_regions(self, block):
```

Comment:
This definitely should not be a method of `GsPoint.`

Reply:
Moved to BlockModel. There is now only one init function for all shapes.